hedera-mirror-rest/viewmodel/nftViewModel.js

```diff
@@ -28,7 +28,7 @@ const EntityId = require('../entityId');
  */
 class NftViewModel {
   constructor(nftModel) {
-    this.account_id = EntityId.fromEncodedId(nftModel.accountId).toString();
+    this.account_id = nftModel.accountId ? EntityId.fromEncodedId(nftModel.accountId).toString() : null;
```

Comment:
Should we push this logic into EntityId so that it can handle null and return null from toString and getEncodedId()?

Reply:
`fromEncodedId` has an `isNullable` parameter, so suggest this change:

```suggestion
    this.account_id = EntityId.fromEncodedId(nftModel.accountId, true).toString();
```

Reply:
I was going to recommend the same thing, just use the isNullable.

Reply:
Yeah, good call, I just missed that bit of logic.  Added the `true` and that works as well, much simpler.